### PR TITLE
Update pe.m3u

### PR DESCRIPTION
**Pull request by sguinetti**
This PR adds 4 channels more, but the periscope streaming is from https://www.pscp.tv/somostv but its broadcast link needs to be improved because it transmits in fragments after 24 hours.

### CODE REVIEW

File: channels/pe.m3u [hunk 1]
```diff
@@ -127,3 +127,11 @@ https://5ee0faac3bbae.streamlock.net/visionnoticias/visionnoticias/playlist.m3u8
 https://5ee0faac3bbae.streamlock.net/visionmusica/visionmusica/playlist.m3u8
 #EXTINF:-1 tvg-id="Willax.pe" tvg-name="Willax" tvg-country="PE" tvg-language="Spanish" tvg-logo="https://graph.facebook.com/willaxtv/picture?width=320&height=320" group-title="General",Willax
 https://query-streamlink.lanesh4d0w.repl.co/iptv-query?streaming-ip=https://www.dailymotion.com/WillaxTV
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="PE" tvg-language="Spanish" tvg-logo="https://graph.facebook.com/rwtelevision/picture?width=320&height=320" group-title="Local",RW Televisión (Tarapoto) [Not 24/7]
+https://tvdatta.com:3952/live/rwtelevisionlive.m3u8
```
Comment (themak1985) [inline channels/pe.m3u:131]: Broken Link

**Review by themak1985 — CHANGES_REQUESTED** [review]

File: channels/pe.m3u (hunk 1 above)
Comment (LaneSh4d0w) [reply channels/pe.m3u:131]: I agree, same on my side.

Comment (MapGuy11) [reply channels/pe.m3u:131]: The link works

**Review by MapGuy11 — APPROVED** [review]

File: channels/pe.m3u (hunk 1 above)
Comment (sguinetti) [reply channels/pe.m3u:131]: The Via Televisió channel had the same problem, so I'm going to put [Not 24/7]. The peruvian issue aobut streaming is very limited compared to other countries.

**Review by themak1985 — APPROVED** [review]

**Pull request merged**